### DISCUSSION TITLE
In a multi screen set-up move active window to the screen where the cursor is

### DISCUSSION THREAD

**Discussion by jvliwanag** [rxhanson/Rectangle#1269] [Ideas]
Consider a multi screen set-up.

Running a keyboard shortcut that say, places a window on the top right corner, would move that window to the top right corner of the screen where it currently is in.

It would be nice to have an option to have the window move first to the screen where the cursor is, and then move/resize on that screen.

**Comment by rxhanson** [comment]
Thanks for the feedback. I don't have plans to add this in myself, but see no reason to turn down a good pull request for it.
